packages/swirl-components/src/components/swirl-form-control/swirl-form-control.tsx

```diff
@@ -242,7 +248,15 @@ export class SwirlFormControl {
             <LabelTag class="form-control__label" onClick={this.onLabelClick}>
               <span class="form-control__label-text" id={this.labelId}>
                 {this.label}
+                {tooltip && (
+                  <span class="form-control__tooltip">
+                    <swirl-tooltip content={tooltip} positioning="fixed">
+                      <swirl-icon-help size={16}></swirl-icon-help>
```

Comment:
Issue: The icon should be focusable to be able to open the tooltip via keyboard. (`tabindex="0"`)